README.md

```diff
@@ -19,6 +19,26 @@ permissions to run Coder.
 - Kubernetes Resosurces: checks that the cluster has the required
 resource types available to run Coder.
 
+## Installation
+
+You can manually download the latest [release](https://github.com/cdr/coder-doctor/releases):
+
+1. Click a release and download the tar file for your operating system
```

Comment:
Suggest using autonumbering (1. 1. 1.) so that we can re-order or add/remove steps without needing to renumber